### DISCUSSION TITLE
Applying force to the model

### DISCUSSION THREAD

**Discussion by Torubara-Oleg** [mfem/mfem#3086] [Q&A] [answered]
Hello everyone!

Can anyone help me about force applying? According to example2 we apply constant force  to each element that is marked with the corresponding attribute. But I don't quite understand if I increase the number of elements on the plane, but I want to keep the force constant for the entire model. Do I need to set a constant force value and divide it by the number of loaded elements or not?

For example, I have a cube. One plane is loaded, and the opposite is fixed. Each plane consists of 8 triangles, if I want to apply a force = 1 newton, then I will have to set the value of the pull_force vector in the corresponding attribute to 1 newton or (1 newton / 8)?

In other words, is the force set in the pull_force vector for the whole model or for each element?

**Comment by mlstowell** [comment, marked as answer]
Hello, @Torubara-Oleg,

Think of it as a force per unit area. The vector field `f` in example 2 is integrated over the boundary so, for a constant `f`, the total force being applied to the surface is the `f * (area of the surface)`. The total force will not depend on the triangulation used to mesh the surface as long as the mesh is fine enough to capture the variation in `f`, if there is any variation.

Best wishes,
Mark
